src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write code here
+const tableHeader = document.querySelector('tr');
+const tableBody = document.querySelector('tbody');
+const toNumber = (string) => {
+  if (typeof string === 'string') {
+    const clearNum = string.replace(/[$, ]/g, '');
+
+    return parseInt(clearNum);
+  }
+};
+
+tableHeader.addEventListener('click', (e) => {
+  const headerChildren = [...tableHeader.children];
+  const columnName = e.target.closest('th');
+  const sortedColumn = headerChildren.indexOf(columnName);
```

Comment:
If 'columnName' is null, 'indexOf' will return -1, which will cause an error when used as an index. Ensure you handle this case appropriately.